tests/getRoutes.test.js

```diff
@@ -0,0 +1,11 @@
+import request from "supertest";
+const url = "http://localhost:3001";
+jest.setTimeout(30000);
+test("if /hacklist returns an array of hacks in the databse", async done => {
+  const response = await request(url).get("/hacklist");
+
+  expect(response.statusCode).toBe(200);
+  expect(response).toBeDefined();
+  expect(Array.isArray(response.body)).toBe(true);
+  done();
```

Comment:
in newer versions of jest, you dont need to call done to do async testing.

https://jestjs.io/docs/en/tutorial-async#asyncawait

by returning a promise (which is what the async keyword does automatically for you), jest will wait for it to resolve before considering the test finished.